doc/developers/contributing.rst

```diff
@@ -14,6 +14,23 @@ adding new algorithms, and the best way to contribute and to help the project
 is to start working on known issues.
 See :ref:`easy_issues` to get started.
 
+.. topic:: **Our community, our values**
+
+    We are a community based on openness and friendly discussions.
+
+    We aspire to treat everybody equal, and value their contributions.
```

Comment:
> > +    We aspire to treat everybody equal, and value their contributions.

> not sure if i'm qualified to make content suggestions,

Well, feedback is always good. And in addition you've been very active
lately.

> but this is rather a grammatical one; shouldn't it be We aspire to
> treat everybody equally?

Indeed. I am afraid that I made a Frenchism "Les hommes naissent et
demeurent en droit" :).